README.md

```diff
@@ -1,3 +1,43 @@
+\frontmatter
+
 # Unfug für Anfänger
 
-_to be written_
+*Unfug für Anfänger* ist ein Buch für alle die an Informatik interessiert sind.
+Es hat mehrere Aufgaben zu erfüllen:
+
+- Referenz für Neueinsteiger in die Welt des modernen Programmierens
+- Neue Ideen für diverse Themen die Praktisch oder einfach nur cool sind
+- Eine FAQ sein
+
+Ob es diese auch schafft bleibt den Lesern überlassen. Humor ist erlaubt und
+auch erwünscht, am Ende geht es darum Spass zu haben; am Programmieren, an der
+Technik, an der Zukunft.
+
+Das Buch ist grob in Teile aufgebaut um den anschein einer Ordnung zu bewahren.
+Das Inhaltsverzeichniss wird bestimmt helfen das zu finden wonach man sucht.
+
+Mit dem sagen wir: Viel Spass beim lesen und treibt ja _UnFUG_.
+
+# Lizenz und letzte Versionen
+
+Dieses Buch ist unter der **CC BY-SA 4.0** Lizenz verfügbar. Dies heißt dass das
+Buch für immer verteilt werden darf, ohne kosten; das man es verändern darf (Und
+diese Veränderung dann auch verteilen darf). Man dürfte das Buch sogar
+verkaufen.  Falls du dass Buch verteilst musst du alle Änderungen markieren,
+sowie die neue Version unter der gleichen Lizenz zugänglich machen. Es muss auch
+auf das Original verlinkt werden, ohne zu implizieren dass die originellen
```

Comment:
This and your previous comment were a try at explaining that just because we made the original document, doesn't mean we agree with the subsequent changes, aka the "Without implied endorsement" clause.